django_irods/storage.py

```diff
@@ -71,23 +72,41 @@ def zipup(self, in_name, out_name):
         :param out_name: the output zipped file name
         :return: None
         """
-        in_bucket, in_path = bucket_and_name(in_name)
+        def chunk_request(zip_archive_file, bucket, key):
+            chunk_size = 1024 * 1024 * 256  # 256MB
```

Comment:
any reason that we might want this to be configurable in a setting?

Reply:
I don't see us changing it but it is easy to update it to allow configuration settings. done.